BrainPortal/lib/boutiques_input_value_fixer.rb

```diff
@@ -0,0 +1,192 @@
+#
+# CBRAIN Project
+#
+# Copyright (C) 2008-2021
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+# This module allows one to fix some of input parameters to specific values
+# The fixed input(s) would no longer be shown to the user in the form.
+#
+# In the descriptor, the spec would look like:
+#
+#    "custom": {
```

Comment:
Please adjust this description block.

* the json example is badly indented
* add a blank line after the json (still commented though) so that rdoc creates a proper code block for it
* some sentences are not finished (`or their values or`)

Reply:
You have not addressed this.